File: src/main/java/bad/robot/radiate/ui/FullScreen.java
package bad.robot.radiate.ui;

import javax.swing.*;
import java.awt.*;

import static java.awt.Color.darkGray;

@Deprecated
class FullScreen implements ScreenMode {
    private final Rectangle bounds;

    public FullScreen(Rectangle bounds) {
        this.bounds = bounds;
    }

    @Override
    public void accept(JFrame frame) {
        frame.setSize(bounds.width, bounds.height);
        frame.setLocation(bounds.x, bounds.y);
        frame.getContentPane().setBackground(darkGray);
        frame.setUndecorated(true);
    }
}

File: src/main/scala/bad/robot/radiate/ui/FullScreen.scala
package bad.robot.radiate.ui

import java.awt.Color.darkGray
import java.awt._
import javax.swing._

class FullScreen(bounds: Rectangle) extends ScreenMode {
  def accept(frame: JFrame) {
    frame.setSize(bounds.width, bounds.height)
    frame.setLocation(bounds.x, bounds.y)
    frame.getContentPane.setBackground(darkGray)
    frame.setUndecorated(true)
  }
}